.github/workflows/clean.yml

```diff
@@ -6,6 +6,7 @@ on:
 
 env:
   FONT_AWESOME_AUTH_TOKEN: ${{ secrets.FONT_AWESOME_AUTH_TOKEN }}
+  FIGMA_TOKEN: ${{ secrets.FIGMA_AUTH_TOKEN }}
```

Comment:
This line is unnecessary for the `clean` workflow?

Reply:
for font awesome as well then?

Reply:
Yeah, good point.

Reply:
Or we need that token for `yarn install` to work?